core/types/wo.go

```diff
@@ -535,6 +535,11 @@ func (wo *WorkObject) SetNumber(val *big.Int, nodeCtx int) {
 	}
 }
 
+func (wo *WorkObject) SetCoinbases(primary common.Address, secondary common.Address) {
```

Comment:
Is `primary` always Quai and `secondary` always Qi? Would be nice to name the args accordingly

Reply:
Not in the workobject. The only difference between primary and secondary is that the coinbase goes to the primary.

But both are needed because blocks may contain transactions from both ledgers. Therefore we need to know where to send fees on each ledger.